examples/sat/sat_tensornetwork.py

```diff
@@ -124,12 +112,11 @@ def sat_count_tn(
       in the clause.
 
   Returns:
-    net: The TensorNetwork corresponding to the norm of the 3SAT TensorNetwork
-      generated by the given clauses.
+    nodes: The set of nodes
   """
-  net1, var_edges1 = sat_tn(clauses)
-  net2, var_edges2 = sat_tn(clauses)
-  sat_net = tensornetwork.TensorNetwork.merge_networks([net1, net2])
+  var_edges1 = sat_tn(clauses)
+  var_edges2 = sat_tn(clauses)
   for edge1, edge2 in zip(var_edges1, var_edges2):
-    sat_net.connect(edge1, edge2)
-  return sat_net
+    edge1 ^ edge2
+  # TODO(chaseriley): Support diconnected SAT graphs.
+  return tn.reachable(var_edges1[0].node1)
```

Comment:
Mypy typecheck failure: error: Incompatible return value type (got "Set[BaseNode]", expected "Set[Node]")

Reply:
See above.

Reply:
Fair enough, except it fails this fails pytype also.

Reply:
Note that travis doesn't run pytest on the examples. But I think it should, since the examples have type annotations.

Reply:
Oh that's a good point. We use to only run pylint and pytype on `tensornetwork/` since `experiments` was an unfixable wasteland, but now that that is nuked we should add it back. I'll do that in a later PR.